todolist/settings.py

```diff
@@ -63,8 +63,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'app_db',
+        'USER': 'app_user',
+        'PASSWORD': '1234',
```

Comment:
Storing database credentials directly in the settings file is not secure. Consider using environment variables or a secrets management tool to handle sensitive information like database passwords.